crates/re_log_types/src/lib.rs

```diff
@@ -253,7 +253,7 @@ pub enum RecordingSource {
     PythonSdk(PythonVersion),
 
     /// The official Rerun Rust Logging SDK
-    RustSdk,
+    RustSdk(String),
```

Comment:
Since `String` could be anything I suggest we give it a descriptive name:

```suggestion
    RustSdk { rust_version: String },
```